README.md

```diff
@@ -1,151 +1,155 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Decode a Morse code] <a name="about-project"></a>
+
+**[Morse-code]** involves the conversion of Morse code signals into corresponding letters or numbers.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+  <ul>
+    <li><a>https://www.ruby-lang.org/en/</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Morse code conversion to words]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- `bundle install`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- cd `decode-a-morse_code`
+- `git clone https://github.com/KAYEMBAJONATHAN/Morse-code.git`
+
+### Install
+
+Install this project with:
+
+- cd morse-code
+- bundle install
+
+### Usage
+
+To run the project execute the following command:
+
+- [N/A]
+
+### Run tests
+
+To run tests, run the following command:
+
+- `rubocop --color`
+
+### Deployment
+
+You can deploy this project using:
+
+- [N/A]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+_[Jonathan]_
+
+- GitHub: [@kayembajonathan](https://github.com/KAYEMBAJONATHAN)
+- LinkedIn: [@kayembajonathan](https://www.linkedin.com/in/jonathan-kayembb56247236/)
+- Twitter: [@JonathanKa97287](@JonathanKa97287)
+
+👤 **Author2**
+_[Abdullah]_
+
+- GitHub: [@AbdullahKhrief](https://github.com/OnlyUnknown)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Decode different codes]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]().
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please show support by staring ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- We would like to thank our Microverse team and our coding partners for their support and guidance.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.

```

Comment:
- [ ] Great effort working on your decode morse project, Currently all the code works fine as expected but i noticed your MIT License link is broken and throwing error 404, I realized that the file name is different from the spelling you have in the README file. You have `./LICENSE` instead of `./LICENCE` as you have it in the file name. Kindly rectify that and be sure the link is working.

MIT error 👇 

![mit](https://github.com/KAYEMBAJONATHAN/Morse-code/assets/78634841/0e6563ff-7654-4e91-8c45-ceb350c329a8)